include/parsing_csv_file.h

```diff
@@ -1 +1,84 @@
-int parsing_csv_file(char *filePath);
+/**
+ * @file parsing_csv_file.h
+ *
+ * @brief Implementation of the CSV parser.
+ */
+#ifndef PARSER_H
+#define PARSER_H
+
+#include <stdio.h>
+#include <errno.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <string.h>
+
+/**
+ * @brief Define a boolean data type.
+ */
+#ifndef Boolean
+    #define Boolean int
```

Comment:
shouldn't this be inline?all # symbols should be inline

Reply:
I don't remember hearing this, but I'll make the change, it looks a little less janky than what I have